Makefile

```diff
@@ -392,6 +392,7 @@ RELEASE_DIR_K8S_MANIFESTS?=$(RELEASE_DIR)/k8s-manifests
 RELEASE_DIR_IMAGES?=$(RELEASE_DIR)/images
 RELEASE_DIR_BIN?=$(RELEASE_DIR)/bin
 RELEASE_WINDOWS_ZIP=$(OUTPUT_DIR)/calico-windows-$(NODE_VER).zip
+RELEASE_HELM_CHART=bin/tigera-operator-$(CALICO_VER).tgz
```

Comment:
currently, charts are produced to match the version passed to them, i.e. `tigera-operator-v3.17.0-1.tgz`. However, I think CALICO_VER is just `v3.17.0`. So this is missing the `-x` chart release indicator.